packages/design-system/src/components/Table/Table.jsx

```diff
@@ -172,6 +171,19 @@ Table.propTypes = {
    * This prop will only be used when the `Table` `scrollable` prop is set and the table width is wider than the viewport.
    */
   scrollableNotice: PropTypes.node,
+  /**
+   * A stackable variation of the table.
+   * When `stackable` is set, `id` or `headers` prop is required in `TableCell`
+   */
+  stackable: PropTypes.bool,
+  /**
+   * Applies responsive styles to vertically stacked rows at different viewpoint sizes.
```

Comment:
```suggestion
   * The viewpoint size at which responsive vertically stacked row styles are applied. Only used when the `stackable` prop is set to `true`.
```

Reply:
These suggestions are great @bernardwang - I think we should change `viewpoint` to `viewport`